apps/mark-scan/backend/src/custom-paper-handler/state_machine.ts

```diff
@@ -150,6 +151,25 @@ export type PaperHandlerStatusEvent =
   | { type: 'RETURN_PREPRINTED_BALLOT' }
   | { type: 'SYSTEM_ADMIN_STARTED_PAPER_HANDLER_DIAGNOSTIC' };
 
+function isEventUserAction(event: EventObject): boolean {
+  return [
+    'VOTER_INITIATED_PRINT',
+    'PAPER_IN_INPUT',
+    'POLL_WORKER_CONFIRMED_INVALIDATED_BALLOT',
+    'VOTER_VALIDATED_BALLOT',
+    'VOTER_INVALIDATED_BALLOT',
+    'PAT_DEVICE_CONNECTED',
+    'PAT_DEVICE_DISCONNECTED',
+    'VOTER_CONFIRMED_PAT_DEVICE_CALIBRATION',
+    'VOTER_CONFIRMED_SESSION_END',
+    'POLL_WORKER_CONFIRMED_BALLOT_BOX_EMPTIED',
+    'RESET',
+    'START_SESSION_WITH_PREPRINTED_BALLOT',
+    'SYSTEM_ADMIN_STARTED_PAPER_HANDLER_DIAGNOSTIC',
+    'PAPER_READY_TO_LOAD', // paper was inserted by a user
+  ].includes(event.type);
```

Comment:
Non-blocker: thinking this could go stale fairly easily - could be worth switching to an exhaustive `Record<X, bool>` or similar to make sure people remember to classify when adding a new event?